docs/_docs/ssl-tls.adoc

```diff
@@ -0,0 +1,193 @@
+= SSL/TLS
+
+This page explains how to configure SSL/TLS encryption between the cluster nodes (server and client) and the clients that connect to your cluster.
+
+== Considerations
+
+All internal connections in the cluster context, as well as cluster's user interaction interfaces, are SSL-enabled. The communication categories are as follows: 
+
+* Between the user and the cluster (node): REST
+* Between the user and the platform clients:
+
+** CLI
```

Comment:
I am not sure that CLI could be named as a platform client. It is a separate application that communicates with the cluster through REST.